python/lsst/ip/diffim/subtractImages.py

```diff
@@ -712,7 +743,12 @@ def _checkMask(mask, sources, badMaskPlanes):
             kept (True) or rejected (False) based on the value of the
             mask plane at its location.
         """
-        badPixelMask = lsst.afw.image.Mask.getPlaneBitMask(badMaskPlanes)
+        setBadMaskPlanes = [
+            maskPlane for maskPlane in badMaskPlanes if maskPlane in mask.getMaskPlaneDict().keys()
```

Comment:
Minor point, but I don't think the `.keys()` is necessary here.

Reply:
Done